_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -156,120 +136,120 @@ The `input` element does not have a semantic role that is a widget role, but sti
 
 #### Failed Example 1
 
-Unknown autocomplete term.
+This `autocomplete` [attribute value][] does not have a [correct autocomplete field][].
 
 ```html
 <label>Username<input autocomplete="badterm"/></label>
 ```
 
 #### Failed Example 2
 
-Term `work` not allowed before `photo`.
+This `autocomplete` [attribute value][] has the `work` token on a [correct autocomplete field][] that does not support this.
 
 ```html
 <label>Photo<input autocomplete="work photo"/></label>
 ```
 
 #### Failed Example 3
 
-Invalid order of terms.
+This `autocomplete` [attribute value][] includes the `work` token before the `shipping` token, instead of the other way around.
 
 ```html
 <label>Email<input autocomplete="work shipping email"/></label>
 ```
 
 #### Failed Example 4
 
-Comma separated rather than space separated list.
+This `autocomplete` [attribute value][] is comma separated instead of space using [ASCII whitespace][].
 
 ```html
 <label>Email<input autocomplete="work,email"/></label>
 ```
 
 #### Failed Example 5
 
-Autocomplete is inappropriate for the type of field.
+This `autocomplete` [attribute value][] is not appropriate for the field. It is not possible to type an e-mail using just numbers.
```

Comment:
Based on JYM's point about making it clear the autocomplete is the error example...

```suggestion
This `autocomplete` [attribute value][] is not appropriate for the field. The form field's implied purpose is to input a quantity (a number) which cannot be a e-mail.
```